README.md

```diff
@@ -12,12 +12,10 @@ Cluster API Provider RKE2 is a combination of 2 provider types, a __Cluster API
 
 ------
 
-## Getting Started
-Follow our [getting started guide](./docs/book/src/getting-started.md) to start creating RKE2 clusters with CAPI.  
+Please see our [book](https://rancher.github.io/cluster-api-provider-rke2/) for in-depth documentation.
```

Comment:
Looks good, but will need an update once https://github.com/rancherlabs/eio/issues/2557 is sorted out